src/network/networkprotocol.h

```diff
@@ -206,6 +206,8 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 
 #define TEXTURENAME_ALLOWED_CHARS "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_.-"
 
+typedef u16 SessionId;
```

Comment:
Does not follow the the other type name characteristics. `session_t`, `peer_t`, `session16` or similar would be appropriate.

Reply:
okay for session_t i like it :)